conf/.codegpt.yaml

```diff
@@ -0,0 +1,28 @@
+git:
+  diff_unified: 3
+  exclude_list: [ ]
+  template_file: ""
+  template_string: ""
+  api_base_url: "https://api.github.com"
+  token: "your github token"
+openai:
+  api_key: "your openai api key"
+  base_url: https://api.openai.com/v1
+  max_tokens: 1500
+  model: gpt-3.5-turbo
+  org_id: ""
+  proxy: "the proxy of openai api"
+  socks: ""
+  temperature: "0.7"
+  timeout: 60s
+output:
+  file: ""
+  lang: en
+server:
+  # "the server ip which you want to bind when running the webhook server"
+  ip: 127.0.0.1
+  # "the port which you want to bind when running the webhook server"
+  port: 8099
+  # the time to sleep when request to the openai to get the comment message each time,
+  # its main purpose is to limit the openai request rate, unit: second
+  sleep_time_second: 40
\ No newline at end of file
```

Comment:
This code patch appears to be a configuration file rather than actual code. Therefore, there are no obvious bugs or security vulnerabilities. However, some improvement suggestions could be:

- Use environment variables instead of hard-coding sensitive information like tokens and API keys.
- Consider adding comments to explain the purpose of each configuration option.
- Verify that the values provided for each configuration option are valid and within expected ranges.
- Use consistent formatting and indentation to improve readability.